README.md

```diff
@@ -23,13 +23,12 @@ To test slimfaas on your local machine by using kubernetes with Docker Desktop,
 ```bash
 git clone https://github.com/AxaFrance/slimfaas.git
 cd slimfaas/demo
-kubectl create namespace slimfaas-demo
-kubectl config set-context --current --namespace=slimfaas-demo
-# Create a custom service account for slimfaas
-# SlimFaas require to ba able to request Kubernetes API
-kubectl apply -f slimfaas-serviceaccount.yml
-# Install slimfaas pod
+# Create slimfaas service account and pods
 kubectl apply -f deployment-slimfaas.yml
+# Expose SlimFaaS service as NodePort or Ingress
+kubectl apply -f slimfaas-nodeport.yml
```

Comment:
By default it could be cool to coment that line to just be able to copy paste command line whithout editing and have something that just work.
#kubectl apply -f slimfaas-nodeport.yml

So ingresss mode woulb be the demo default mode

Reply:
Good point about having one commented by default, but it probably should be the `Ingress` line that is commented then. Ingress is arguably more common in production setups, but you need to have an `IngressController` installed for that (it also defines the port you can reach SlimFaaS on). For instance, I run a [k3s](https://k3s.io/) installation, which has [Traefik Proxy](https://traefik.io/traefik/) installed as its `IngressController` out of the box. To make the example more robust, the `NodePort` config can be suggested, which will work with any setup, including a simple Docker Desktop installation. People who have an `IngressController` set up will be knowledgeable enough to skip the `NodePort` (to avoid opening additional ports) and customise the `Ingress` instead.

I've now changed the PR to comment out the `slimfaas-ingress` command.